src/main/drivers/gyro_sync.c

```diff
@@ -14,13 +14,28 @@
 #include "accgyro.h"
 #include "gyro_sync.h"
 
+#include "common/axis.h"
+
+#include "sensors/gyro.h"
+
 static uint8_t mpuDividerDrops;
+static volatile bool pidDataReady;
```

Comment:
This is a misnomer. It's not about whether PID data is ready - there is no PID data that needs to be ready. `isPidScheduledToRun` is a better name.